### ISSUE TITLE
Can't register a new account if integrated captcha is active

### ISSUE THREAD

**Issue by MechDR** [SimpleMachines/SMF#5582]
#### Description
If the default integrated (standard, not ReCAPTCHA) is active and you try to register a new account (a new user), it throws a "You are not allowed to access this section" error. Disabling the integrated captcha solves the problem.

https://www.simplemachines.org/community/index.php?topic=547871.0

### Steps to reproduce
1. Enable integrated captcha on registration.
2. Try to register a new account.

### Environment (complete as necessary)
- Version/Git revision: RC2
- Database Type: MySQL (MyISAM)
- Database Version: 5.6.43
- PHP Version: 7.2.16

### Additional information/references

**Comment by VBGAMER45** [comment]
Probably something in sources/Subs-editor.php

> 	$verification_errors = array();
	// Start with any testing.
	if ($do_test)
	{
		// This cannot happen!
		if (!isset($_SESSION[$verificationOptions['id'] . '_vv']['count']))
			fatal_lang_error('no_access', false);
		// ... nor this!
		if ($thisVerification['number_questions'] && (!isset($_SESSION[$verificationOptions['id'] . '_vv']['q']) || !isset($_REQUEST[$verificationOptions['id'] . '_vv']['q'])))
			fatal_lang_error('no_access', false);
		// Hmm, it's requested but not actually declared. This shouldn't happen.
		if ($thisVerification['empty_field'] && empty($_SESSION[$verificationOptions['id'] . '_vv']['empty_field']))
			fatal_lang_error('no_access', false);

>

**Comment by live627** [comment]
@MechDR can you please replace all three instances of ` fatal_lang_error('no_access', false);` with ` fatal_lang_error('no_access');` (should be around line 2074) and then report the logged error? What you come up with will be all I can go on because I cannot reproduce this one.

**Comment by MechDR** [comment]
I can't reproduce the error now. Tried changing `fatal_lang_error('no_access', false);` to `fatal_lang_error('no_access');`, but there was no problem registering a new user with either the original Subs-editor.php or the modified one. Even tried it a few times, with different usernames... no problem now.

Though I did disable the integrated captcha and then re-enabled it to do this test again, so I have no idea if maybe disabling it and then re-enabling it did the trick (didn't try it that way before).
